DirectEmployers/MyJobs
# [ST-4432] - fixes a bug where reached_job_source appeared first instead of last

## What's this PR do?
as described in title

## Screenshots (if appropriate)
column orders will go back to what it was before:
![image](https://user-images.githubusercontent.com/27963853/30437515-cee65ce8-993c-11e7-92df-cb4e47aff063.png)

## How should this be manually tested?
run master branch locally and verify `reached_job_source` no longer appears first 

## Any background context you want to provide?
This commit should not be backported to QC and Staging.
